Formula/go.rb

```diff
@@ -47,11 +37,11 @@ class Go < Formula
   depends_on :macos => :mountain_lion
 
   # Should use the last stable binary release to bootstrap.
-  # More explicitly, leave this at 1.7 when 1.7.1 is released.
+  # More explicitly, leave this at 1.8 when 1.8.1 is released.
```

Comment:
I believe they quietly cut a new release, and we just never moved back to 1.4 because 1.7 wasn't causing any issues and changes to the `go` formula are rarely simple 😄, but I could be misremembering. I know there was talk of cutting a new 1.4 release simply for the sake of bootstrapping at least.

But yes, you're correct that having a `gobootstrap` or such from source would've allowed us to patch the 1.4 build failure at the time and keep using that branch for the process. The downside is that building even Go 1.4 is not ultra-quick in comparison to the existing process, and there hasn't been many clear upsides in the past.

> In any case, assuming for the sake of argument that I accept that this use-the-prior-version-to-bootstrap approach is useful, why 1.7 not 1.7.5?

No reason at all. I simply wanted to avoid the confusion & hassle of people updating the bootstrap for every minor release. Less moving parts, less potential for breakage, etc etc.